src/table.js

```diff
@@ -846,6 +901,10 @@ Table.prototype.createExtractJob = function(destination, options, callback) {
     delete options.jobPrefix;
   }
 
+  if (this.location) {
```

Comment:
Why is `options.location` not accepted here, but it is in, e.g. `createCopyFromJob()`?

Reply:
`createCopyJob` still accepts `metadata.location`. Also, `metadata.location` is documented for `copy`, `copyFrom`, and `createCopyJob`. I'm only mentioning it in this thread and not as line comments, because I'm not sure if I'm just confused and they are correct, or if they need to be removed.

Reply:
They needed to be removed. Which I have done